src/wallet/wallet.cpp

```diff
@@ -1105,6 +1105,9 @@ void CWallet::BlockDisconnected(const CBlock& block, int height)
 void CWallet::UpdatedBlockTip(bool is_ibd)
 {
     m_best_block_time = GetTime();
+    auto locked_chain = chain().lock();
```

Comment:
In commit "Add m_is_ibd in CWallet" (dca4246dac0f4fa6369a71ee787b2bf1f6415c6d)

Calling chain lock not unnecessary here (but harmless)